<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Explorer</class>
 <widget class="QDialog" name="Explorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeView" name="explorer_directoryViewer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>511</width>
     <height>420</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="explorer_fileViewer">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>20</y>
     <width>331</width>
     <height>420</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="explorer_saveas_lineedit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[YOUR_LAYOUT_NAME]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="explorer_saveas_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This will save the current layout under the above selected directory (i.e. C:/) followed by the name you chose in the text-field to the right [YOUR_LAYOUT_NAME].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </widget>
  <widget class="QPushButton" name="explorer_save_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="explorer_import_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="explorer_export_button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>480</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="explorer_cancel_button">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>480</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="add_to_my_layouts_checkbox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>510</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to My Layouts</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.sock</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>explorer_save_button</tabstop>
  <tabstop>explorer_saveas_button</tabstop>
  <tabstop>explorer_saveas_lineedit</tabstop>
  <tabstop>explorer_import_button</tabstop>
  <tabstop>explorer_export_button</tabstop>
  <tabstop>explorer_directoryViewer</tabstop>
  <tabstop>explorer_fileViewer</tabstop>
  <tabstop>explorer_cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
